ui/pages/extensions.js

```diff
@@ -266,11 +266,227 @@ const MesheryDesignEmbedExtension = () => {
+            Layer5 Cloud
+          </Typography>
+
+          <FrontSideDescription variant="body">
+            <Layer5CloudLogo />A comprehensive platform offering identity and collaboration
+            services, private catalogs, GitOps, and multi-Meshery management. Leverage its
+            extensible authorization framework and organizational hierarchy for streamlined cloud
+            infrastructure management.
+          </FrontSideDescription>
+
+          <div style={{ textAlign: 'right' }}>
+            <Button
+              variant="contained"
+              color="primary"
+              data-testid="layer5-cloud-learn-more-btn"
+              onClick={(e) => handleLearnMore(e)}
+            >
+              Learn More
+            </Button>
+          </div>
+        </CardContainer>
+      </Grid>
+    </>
+  );
+};
+
+const HelmPluginLogo = () => {
+  return (
+    <img
+      style={{
+        paddingRight: '1rem',
+        height: 'auto',
+        width: 'auto',
+        maxWidth: '120px',
+        maxHeight: '75px',
+      }}
+      src="/static/img/helm-icon.svg"
+      data-testid="helm-plugin-image"
+    />
+  );
+};
+
+const HelmPluginExtension = () => {
+  const handleLearnMore = (e) => {
+    window.open('https://meshery.io/extensions/helm-kanvas-snapshot', '_blank');
+    e.stopPropagation();
+  };
+
+  return (
+    <>
+      <Grid item {...LARGE_6_MED_12_GRID_STYLE}>
+        <CardContainer>
+          <Typography data-testid="helm-plugin-heading" variant="h5" component="div">
+            Helm Plugin for Kanvas Snapshot
+          </Typography>
+
+          <FrontSideDescription variant="body">
+            <HelmPluginLogo />
+            Generate visual snapshots of your Helm charts directly from the command line. Helm
+            charts into Kanvas Snapshots with support for both packaged and unpackaged charts.
+            Receive snapshots via email or terminal URL display.
+          </FrontSideDescription>
+
+          <div style={{ textAlign: 'right' }}>
+            <Button
+              variant="contained"
+              color="primary"
+              data-testid="helm-plugin-learn-more-btn"
+              onClick={(e) => handleLearnMore(e)}
+            >
+              Learn More
+            </Button>
+          </div>
+        </CardContainer>
+      </Grid>
+    </>
+  );
+};
+
+const KubectlPluginLogo = () => {
+  return (
+    <img
+      style={{
+        paddingRight: '1rem',
+        height: 'auto',
+        width: 'auto',
+        maxWidth: '120px',
+        maxHeight: '75px',
+      }}
+      src="/static/img/kubectl-icon.svg"
+      data-testid="kubectl-plugin-image"
+    />
+  );
+};
+
+const KubectlPluginExtension = () => {
+  const handleLearnMore = (e) => {
+    window.open('https://docs.meshery.io/extensions/kubectl-kanvas-snapshot', '_blank');
+    e.stopPropagation();
+  };
+
+  return (
+    <>
+      <Grid item {...LARGE_6_MED_12_GRID_STYLE}>
```

Comment:
Where is "LARGE_6_MED_12_GRID_STYLE" coming from? 

Is this used anywhere else in Meshery UI? @Karan-Palan